README.md

```diff
@@ -13,4 +13,5 @@ This repository contains samples for creating an Azure Marketplace Kubernetes of
 
 ## Samples in this repo
 
-- azure-vote: The Azure Vote application is a simple sample used throughout the Azure Container Service (AKS) documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage.
+- k8s-offer-azure-vote: The Azure Vote application is a simple sample used throughout the Kubernetes Applications Marketplace documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage.
```

Comment:
Not related to this PR change. While reviewing your PR, observed this. Can you please update the description content above?
Please change 'This repository contains samples for creating an Azure Marketplace Kubernetes offer.' to 'This repository contains samples for creating Kubernetes Applications in Azure Marketplace.'

Reply:
Marketplace doesn't have a offer with name Kubernetes Offer and so good to update the description.